src/App.css

```diff
@@ -1,38 +1,119 @@
-.App {
+.container {
```

Comment:
this file contains ALL the styles of the application, this is not recommended because as the app grows this file keeps getting bigger and bigger and this makes things hard to maintain, also using CSS files can cause style conflicts, an alternative is to use styled-components

The `Module 2 Styles in React / CSS in JS ` covers the topic in depth,